app/code/community/HE/TwoFactorAuth/etc/config.xml

```diff
@@ -2,7 +2,7 @@
 <config>
     <modules>
         <HE_TwoFactorAuth>
-            <version>1.1.0</version>
+            <version>1.1.1</version>
```

Comment:
Right, this will be done from the release branch

I believe the procedure is 

feature -> develop 

branch release/v1.1.2 out of develop

release/v1.1.2 - bump version number

release/v1.1.2 -> develop

release/v1.1.2 -> master